### ISSUE TITLE
Unknown DDOC macros are 

### ISSUE THREAD

**Issue by Jebbs** [Pure-D/serve-d#8]
Currently, if the documentation parser encounters a macro that isn't a predefined one, what ever is inside that macro is left out of the documentation popup.

Perhaps not the best solution, but I wonder if it would be better to considered all unknown macros as being equal to `$0` so that the text being passed to them is at least visible. Some projects will need to rework macros to take this into account (perhaps the text they are displaying is not the first macro param), but at least it is a way to get text to appear in the documentation popup.

**Comment by WebFreak001** [comment]
yeah I noticed that but I'm not sure if that is possible with the ddoc library I'm using right now (https://code.dlang.org/packages/libddoc).

From what I tried it looked like it isn't possible but I don't know for certain because there isn't any example or code showing how it works other than the unittests
